lib/aws/iam.js

```diff
@@ -0,0 +1,78 @@
+var inherits = require('util').inherits;
```

Comment:
Can we create another folder like 'aws-promise' under 'lib' to separate the files implemented by promise instead of using xxx2.js?